gapic-showcase-test/showcase-protos/google/api/client.proto

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "google.golang.org/genproto/googleapis/api/annotations;annotations";
+option java_multiple_files = true;
+option java_outer_classname = "ClientProto";
+option java_package = "com.google.api";
+option objc_class_prefix = "GAPI";
+
+
+// Information about the API as a whole.
+// This is generally used for client library packaging and documentation, and
+// only needs to be set when correct values can not be inferred from the proto
+// package.
+//
+// For example, Pub/Sub sets this to:
+//
+//     option (google.api.package) = {
+//       title: "Pub/Sub"
+//       namespace: ["Google", "Cloud"]
+//       version: "v1"
+//     };
+//
+// If this is set, then the `title`, `namespace`, and `version` fields
+// should all be set.
+message Package {
```

Comment:
Is this the same file that is available at `shared/api-common-protos/google/api/annotations.proto`?

Reply:
Yes, but that stuff was moved. (i.e. most of the things in `annotations.proto` were moved to `client.proto` a few weeks ago).